### ISSUE TITLE
Hawku Tablet driver dont work with G640s

### ISSUE THREAD

**Issue by Minter1461** [hawku/TabletDriver#909]
## Hardware/software information
Windows Version: 10

Tablet Driver version: 0.2.3

Tablet model: XP-Pen G640s

Helloi have the problem that the driver recognize the tablet but when i move the pen it doesnt respond.#
Please help 
![Bild_2020-12-17_231259](https://user-images.githubusercontent.com/76185782/102550019-686e7c80-40bd-11eb-80be-0a5dd1fca8e8.png)

**Comment by tallisbackatcoding** [comment]
same with ctl 472

**Comment by Mrturtlenugget** [comment]
yeah me too and i am assuming you too have uninstalled other tablet drivers i use Xp pen star G430s too
